packages/junokit/sample-project/scripts/sample-script.js

```diff
@@ -7,6 +7,32 @@ async function run() {
   await contract.setUpclient();
   await contract.parseSchema();
 
+  const junoswap_contract = new ExternalContract("junoswap");
+  await junoswap_contract.setUpclient();
+
+  // query external contract
+  const ext_query_response = await junoswap_contract.query(
+    "balance",
+    {
+      address: contract_owner.account.address
+    }
+  );
+  console.log(ext_query_response);
```

Comment:
i think we should add this in junokit-templates and keep the initial boilerplate easier to understand.

Reply:
sure, we can move it to a template which is more complex example. For example https://github.com/arufa-research/junokit-templates/issues/5